### ISSUE TITLE
Why is DBeaver much slower on Windows than Linux?

### ISSUE THREAD

**Issue by lemon-juice** [dbeaver/dbeaver#6821]
I am very surprised to realize that DBeaver is much faster and responsive on Linux than on Windows. I've just installed DBeaver on Linux Mint on the same computer that I always use DBeaver on Windows. It looks like it's not database operations that are faster on Linux (I used local connections for comparison) but the rendering of UI elements: opening tabs with table properties, switching between tabs, opening tabs with SQL editors, etc. All these operations that require some 1 or 2-second delay on Windows are much faster on Linux - around 2-3 times faster. Overall, on Linux DBeaver is much snappier.

What might be the reason for this? Is Java less efficient on Windows? Is there a remedy for this?

**Comment by serge-rider** [comment]
I believe it is mostly because of fancy theme rendering. You could try to switch to Classic theme on Windows (Preferences->General->Appearance).
This is very dependent on a particular machine. But sometimes it helps a lot.

**Comment by lemon-juice** [comment]
Thank for responding, I've already tried different theme settings and they didn't make any difference. However, I opened the Windows Task Manager and observed the processes that used most cpu when opening new SQL Editor tabs in DBeaver and I found the culprit - it was [4t Tray Minimizer](http://www.4t-niagara.com/tray.html) application that I had installed on my system. Once I closed it then DBeaver became as fast as the one on Linux. I wasn't expecting the minimizer app to be so invasive, I could understand it might cause some delays when opening new windows but not when opening tabs within a Java application.

Well, it's good to know this is not a DBeaver problem, a man learns something new all the time...
